workers/interfaces/WorkerUtils/navigator/008.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<title>readonlyness of members of Navigator in strict mode</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+async_test(function() {
+  var worker = new Worker('008.js');
+  worker.onmessage = this.step_func(function(e) {
+    assert_equals(e.data, '');
+    this.done();
+  });
+});
+</script>
```

Comment:
Newline at the end of the file, but please also use https://web-platform-tests.org/writing-tests/testharness.html#dedicated-worker-tests-worker-js instead. That'll be much clearer.